prisma/seed.ts

```diff
@@ -24,12 +24,15 @@ type MainSchemaType = {
   inviteOrContactLink?: string
   igHeadTeleUsername: string
   igHeadPreferredName?: string
+  otherMembers?: string
```

Comment:
not very sure what this field is for. If there are multiple other members what happens? Is only one of the other members included here?

Reply:
Terrible naming choice - each IG must have at least one head, but if there are more IG Heads/members of the ExCo who need booking permissions, their names go in that field